src/app/Dashboard/LayoutTemplateUploadModal.tsx

```diff
@@ -162,7 +162,7 @@ export const LayoutTemplateUploadModal: React.FC<LayoutTemplateUploadModalProps>
                   onSingleSuccess(
                     fileUpload.file.name,
                     <Text component="p" style={{ color: 'var(--pf-global--warning-color--200)' }}>
-                      Warning: To see this template in the template picker, make sure the Cryostat Feature Level is set
+                      Warning: To see this template in the template picker, make sure the Cryostat Feature level is set
```

Comment:
I think here, "Feature Level" was treated as a capitalized term when it should not have been, so this should be `feature level`